src/test/util_tests.cpp

```diff
@@ -2456,4 +2456,45 @@ BOOST_AUTO_TEST_CASE(remove_prefix)
     BOOST_CHECK_EQUAL(RemovePrefix("", ""), "");
 }
 
+BOOST_AUTO_TEST_CASE(util_ParseByteUnits)
+{
+    // default multiplier is MiB
+    BOOST_CHECK_EQUAL(ParseByteUnits("1").value_or(0), 1ULL << 20UL);
+    BOOST_CHECK_EQUAL(ParseByteUnits("10").value_or(0), 10ULL << 20UL);
+
+    BOOST_CHECK_EQUAL(ParseByteUnits("1k").value_or(0), 1000ULL);
+    BOOST_CHECK_EQUAL(ParseByteUnits("1K").value_or(0), 1ULL << 10);
+
+    BOOST_CHECK_EQUAL(ParseByteUnits("2m").value_or(0), 2ULL * 1000 * 1000);
+    BOOST_CHECK_EQUAL(ParseByteUnits("2M").value_or(0), 2ULL << 20);
+
+    BOOST_CHECK_EQUAL(ParseByteUnits("3g").value_or(0), 3ULL * 1000 * 1000 * 1000);
+    BOOST_CHECK_EQUAL(ParseByteUnits("3G").value_or(0), 3ULL << 30);
+
+    BOOST_CHECK_EQUAL(ParseByteUnits("4t").value_or(0), 4ULL * 1000 * 1000 * 1000 * 1000);
+    BOOST_CHECK_EQUAL(ParseByteUnits("4T").value_or(0), 4ULL << 40);
+
+    // override default multiplier
+    BOOST_CHECK_EQUAL(ParseByteUnits("5", 1ULL << 10).value_or(0), 5ULL << 10);
```

Comment:
In commit "util: ParseByteUnits - Parse a string with suffix unit [k|K] [m|M] [g|G] [t|T]" (c5d4a59cf91f54a2027851b32cf83cfcb04cd210)

Would be good to check behavior with 0 multiplier to make sure it doesn't accidentally change in the future.

Reply:
Added `0` multiplier test